src/front/app/components/challengeTopics/ChallengeTopics.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ChallengeTopic from './ChallengeTopic';
+
+let ChallengeTopics = ({challengeTopics = [], getChallengeTopics}) => {
+    if(!challengeTopics.length) {
+        getChallengeTopics();
+    }
+
+    return (
+        <ul>
+            {challengeTopics.map((challengeTopic, index) => {
+                return (
+                    < ChallengeTopic key={index} challengeTopic={challengeTopic}/>
+                );
+            })}
+        </ul>
+    );
+};
+
+const mapStateToProps = (state) => {
+    return {
+        challengeTopics: state.app.challengeTopics
+    };
+};
+
+const mapDispatchToProps = (dispatch) => {
+    return {
+        getChallengeTopics: () => {
```

Comment:
you don't realy need arrow here, you can use short object function field definition
```js
{
     getChallengeTopics() {
         /// do stuff
     }
}

```